saleor/graphql/notifications/mutations/external_notification_trigger.py

```diff
@@ -0,0 +1,142 @@
+import json
+
+import graphene
+
+from saleor.account.models import User
+from saleor.checkout.models import Checkout
+from saleor.core.exceptions import PermissionDenied
+from saleor.core.permissions import (
+    CheckoutPermissions,
+    OrderPermissions,
+    ProductPermissions,
+)
+from saleor.graphql.core.types.common import ExternalNotificationError
+from saleor.invoice.models import Invoice
+from saleor.order.models import Fulfillment, Order
+from saleor.product.models import Product, ProductVariant
+
+from ....core.permissions import AccountPermissions
+from ....graphql.utils import resolve_global_ids_to_primary_keys
+from ....webhook.payloads import (
+    generate_checkout_payload,
+    generate_customer_payload,
+    generate_fulfillment_lines_payload,
+    generate_invoice_payload,
+    generate_order_payload,
+    generate_product_payload,
+    generate_product_variant_payload,
+)
+from ...core.mutations import BaseMutation
+from ..inputs import ExternalNotificationTriggerInput
+
+
+class ExternalNotificationTrigger(BaseMutation):
+    PK__IN = "pk__in"
+    PAYLOAD_MAPPING = {
```

Comment:
It doesn't have to be a class variable, we could move it outside.

Reply:
What's the difference? Only `ExternalNotificationTrigger` gonna use that mapping so IMO should be kept within that class.

Reply:
Readability - the entire class gets longer for no specific reason. Also, this adds an additional indentation level to this dictionary.